### PR TITLE
feat(datepicker): Allow to disable specific dates

### LINKED ISSUE #1821: feat(datepicker): add functionality of applying array of enabled dates to the datepicker component
I have an array of enabled dates that I want to apply to the date picker component. Currently there only seems to be an Input for dateDisabled which does the exact opposite or what I am after. Is there a way to get ng2-bootstraps datepicker to only enable a set of dates from an array?

### LINKED ISSUE #5028: [datepicker ]How Can we disable/enable multiple dates like 2019-01-28,2019-01-31.
I am using the latest one. But unable to disable/enable specific dates like 2019-01-28,2019-01-31

### PR DESCRIPTION
**Pull request by frederikprijck**
# PR Checklist
Before creating new PR, please take a look at checklist below to make sure that you've done everything that needs to be done before we can merge it.

 - [x] read and followed the [CONTRIBUTING.md](https://github.com/valor-software/ngx-bootstrap/blob/development/CONTRIBUTING.md) guide.
 - [x] built and tested the changes locally.
 - [ ] added/updated tests.
 - [ ] added/updated API documentation.
 - [ ] added/updated demos.

closes #5028  
related to #4993 
closes #1821

### CODE REVIEW

File: demo/src/app/components/+datepicker/datepicker-section.list.ts [hunk 1]
```diff
@@ -173,6 +174,15 @@ export const demoComponentContent: ContentSection[] = [
           Sunday is considered the first day of the week and thus has the value 0</p>`,
         outlet: DemoDatepickerDaysDisabledComponent
       },
+      {
+        title: 'Dates disabled',
+        anchor: 'dates-disabled',
+        component: require('!!raw-loader?lang=typescript!./demos/disable-dates/disable-dates.ts'),
+        html: require('!!raw-loader?lang=markup!./demos/disable-dates/disable-dates.html'),
+        description: `
+          Decent description has to come here!!!`,
```
Comment (ludmilanesvitiy) [inline demo/src/app/components/+datepicker/datepicker-section.list.ts]: @frederikprijck Could you, please add appropriate description here?

Comment (frederikprijck) [reply demo/src/app/components/+datepicker/datepicker-section.list.ts]: I've updated the description.

**Comment by frederikprijck** [comment]
@Domainv Anything you want from me before this can get merged? I'd love to resolve it so we can get this merged!

**Comment by Domainv** [comment]
@frederikprijck Needs to add a unit test for the functionality

**Review by Domainv — DISMISSED** [review]

**Review by Domainv — DISMISSED** [review]

**Comment by ludmilanesvitiy** [comment]
Demo tested on latest Chrome, FF - looks good. Code reviewed, looks good. Related links added.

**Comment by ludmilanesvitiy** [comment]
Retested functionality with Daterangepicker demo, checked disabledDates param with inline datepicker, looks good.

**Review by valorkin — APPROVED** [review]

**Pull request merged**